recipes/yaha/meta.yaml

```diff
@@ -0,0 +1,24 @@
+about:
+  home: 'https://github.com/GregoryFaust/yaha'
+  license: 'GPL'
+  summary: 'yaha is an open source, flexible, sensitive and accurate DNA aligner designed for single-end reads'
+
+build:
+  number: 0
+  skip: #[not py27 and osx]
+
+package:
+  name: yaha
+  version: '0.1.83'
+
+requirements:
+  build:
```

Comment:
libgcc needs to be in the runtime dependencies. Please have a look at our guidelines.